ext/liquid_c/parser.c

```diff
@@ -2,7 +2,7 @@
 #include "parser.h"
 #include "lexer.h"
 
-static VALUE cLiquidRangeLookup, cLiquidVariableLookup, cRange, symBlank, symEmpty;
+static VALUE cLiquidRangeLookup, cLiquidVariableLookup, cRange, vLiquidExpressionLiterals;
```

Comment:
What's the naming convention for values? I picked `v` arbitrarily, not sure what's commonly used.